build

```diff
@@ -97,25 +95,26 @@ elif [ -n "$TARGET" ]; then
 fi
 
 
-build_args=""
+build_args=()
+
+while read -r line && test "${line}"; do
```

Comment:
The `test` here seems to be missing an operator (`-n`?)

Reply:
this isn't required, however I've added it to be explicit we're testing for string length:

```sh
$ test ""; echo $?
1
$ test "contents"; echo $?
0
```